apps/cookbook/src/app/examples/data-table-example/examples/card.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from '@angular/core';
+import { table_example_data } from '../table_example_data';
+
+interface TableIF {
```

Comment:
Consider something like 
```suggestion
interface Person {
```